draft-ietf-tls-tls13.md

```diff
@@ -3708,13 +3693,9 @@ the TLS PRF. This document replaces the PRF with HKDF, thus requiring
 a new construction. The exporter interface remains the same, however
 the value is computed as:
 
-    HKDF-Expand-Label(HKDF-Extract(0, exporter_secret),
+    HKDF-Expand-Label(exporter_secret
                       label, context_value, length)
```

Comment:
nit: I suggest "key_length" instead, here, for consistency with the previous use of HKDF-Expand-Label() a bit above.